tools/formatter.md

```diff
@@ -1,7 +1,16 @@
 ## Code formatter
 
-Deno ships with a built in code formatter that auto-formats TypeScript,
-JavaScript, Markdown, JSON, and JSONC code.
+Deno ships with a built in code formatter that will auto-format the following files:
+
+| File Type  | Extension          |
+| ---------- | ------------------ |
+| JavaScript | `.js`              |
+| TypeScript | `.ts`              |
+| Markdown   | `.md`, `.markdown` |
+| JSON       | `.json`            |
+| JSONC      | `.jsonc`           |
```

Comment:
Also supports JSX (`.jsx` and `.tsx`).

Reply:
Would you class that as

|JSX|`.jsx`|
|TSX|`.tsx`|

Or would that fall under JS/TS


|JavaScript|`.js`,`.jsx`|
|TypeScript|`.js`,`.tsx`|

?

Reply:
I suggest the first option